### DISCUSSION TITLE
Erro 422 Api Itau

### DISCUSSION THREAD

**Discussion by Ptubbie** [viniciussanchez/RESTRequest4Delphi#221] [General]
Quando recebo um erro 422 da Api Itau o rest não mostra o body da Api ele gera uma exceção, e fico sem saber qual o motivo do erro, como faço pra tratar esse erro 422?
Consegue me ajudar?

**Comment by viniciussanchez** [comment]
Boa tarde... da para tratar pelo status code pegando o response da requisição e verificando a respectiva propriedade...

**Comment by Ptubbie** [comment]
Boa tarde,
Como faço pra tratar esse erro, mesmo sendo erro 400 a api manda um json com a informação que esta errado, nessa caso e cpf invalido, n
Project Servidor.exe raised exception class EIdHTTPProtocolException with message 'HTTP/1.1 400 '.

gostaria de receber o json que api esta enviado junto com codigo 400
{
  "errors": [
    {
      "code": "invalid_object",
      "description": "O CPF/CNPJ informado é inválido."
    }
  ]
}
![Erro400](https://github.com/user-attachments/assets/580846a6-6f7a-4eb3-802e-c95005ee533a)
